index.html

```diff
@@ -264,350 +274,353 @@ <h2>Intro Section of the Document</h2>
+        </section>
+        
+        <section id="req-openrev">
+          <h3>Reviewer Access</h3>
+          <p><span class="rfc2119-assertion" id="submission-requirements-openrev">Reviewers MUST have access to the binding document and to the protocol or media type specification (what the binding specifies).</span></p>
+        </section>
+        
+        <section id="req-summ">
+          <h3>Summary Document</h3>
+          <p><span class="rfc2119-assertion" id="submission-requirements-summary-document">The submitter MUST fill in the GitHub form provided by the custodian to generate the summary document, which is hosted by the custodian together with the registry.</span> This form contains the following:</p>
+          <ul>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-abstract">Abstract - It MUST contain an abstract with the following information</span>:<ul>
+            <li>What is the content of the binding about, e.g., what is this protocol?</li>
+            <li>Who should use it?</li>
+            <li>For what purpose(s) should it be used, e.g., monitoring, process control? This SHOULD use terminology of the submitter, i.e., the custodian does not provide definitions for this.</li>
             </ul>
-          </dd>
-          <dt>
-            <dfn id="req-content">Req-Content</dfn>
-          </dt>
-          <dd>
-            <p>The binding MUST contain the following sections in the order presented below. The binding CAN contain other sections anywhere, including between the required ones. The submitters are encouraged to look at the existing submissions. There MUST be at least one operation mapped to a protocol message/vocabulary term. The submitter SHOULD use the table template provided in the document for the vocabulary.</p>
-            <ul>
-              <li>Introduction</li>
-              <li>Binding Content:<ul>
-              <li>URL Format</li>
-              <li>Form Vocabulary Definition as Table</li>
-              <li>Default and possible mappings to operations as a Table</li>
-              </ul>
-              </li>
-              <li>Examples</li>
+            </li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-examples">Examples - It SHOULD contain examples (can be one) TDs or TMs demonstrating the use of the binding</span></li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-access-usage">It MUST contain Access/Usage restrictions about the binding, protocol, implementation, etc., using the terminology and/or documents of the submitter.</span> A non-exhaustive list of examples of restrictions:
+              <p class="ednote">[DP] the list is not correctly nested but was already that way in the <a href="https://github.com/w3c/wot-bindings-registry/blob/72f86e706a7292b1419e4c088fb8e1c16e1a4e01/requirements.md#req-summ">original MD document</a>.</p>
```

Comment:
This should be nested another level no?

Reply:
Yes, it should be another level but to me, it is not clear when the nesting stops...

Reply:
Oh I see now... It should stop after `<li>Making a statement about your product&#39;s supporting that binding</li>`

Reply:
fixed, see https://github.com/w3c/wot-bindings-registry/pull/17/commits/339393d1106730c909899bc30f7bfcbf9861d036